command/s3_put.go

```diff
@@ -234,10 +246,6 @@ retryLoop:
 					remoteName = fmt.Sprintf("%s%s", s3pc.RemoteFile, fname)
 				}
 
-				auth := &aws.Auth{
-					AccessKey: s3pc.AwsKey,
-					SecretKey: s3pc.AwsSecret,
-				}
 				s3URL := url.URL{
 					Scheme: "s3",
 					Host:   s3pc.Bucket,
```

Comment:
unrelated to your CR, but 259 says "important to *not* wrap this error" and then wraps it

Reply:
the not wrapping it was an artifact of the old retry mechanism, which is no longer relevant, so we've removed it.